variables.tf

```diff
@@ -29,4 +29,32 @@ variable "availability_zones" {
   type        = list(any)
   description = "AZ of the region"
   default     = ["eu-west-2a", "eu-west-2b", "eu-west-2c"]
+}
+########
+variable "rds_vpc_id" {
```

Comment:
rds_vpc_id, rds_public_subnets, rds_public_subnet_group can be removed

Reply:
resolved